dipy/segment/metric.py

```diff
@@ -1,15 +1,23 @@
 from dipy.segment.featurespeed import (Feature,
                                        IdentityFeature,
-                                       CenterOfMassFeature)
+                                       ResampleFeature,
+                                       CenterOfMassFeature,
+                                       MidpointFeature,
+                                       ArcLengthFeature,
+                                       VectorBetweenEndpointsFeature)
 
 from dipy.segment.metricspeed import (Metric,
                                       SumPointwiseEuclideanMetric,
                                       AveragePointwiseEuclideanMetric,
-                                      MinimumAverageDirectFlipMetric)
+                                      MinimumAverageDirectFlipMetric,
+                                      CosineMetric)
 
 from dipy.segment.metricspeed import (dist,
                                       distance_matrix)
 
+# Creates aliases
+EuclideanMetric = SumPointwiseEuclideanMetric
```

Comment:
No. Depending on the context, I find it clearer. Particularly, when comparing scalars or N-dimensional points. I think it makes more sense to use the term `EuclideanMetric` when computing Euclidean distance between two points and use `SumPointwiseEuclideanMetric` when comparing two sequences. Since `SumPointwiseEuclideanMetric` is generic it can easily handles the scalar and single Nd point cases.

For example, somewhere in the tutorial about extending features, we have to compare scalars: `metric = EuclideanMetric(feature=ArcLengthFeature())`.